src/interactors/interactors.test.js

```diff
@@ -29,4 +30,18 @@ describe('Interactors', () => {
     await registerDevice.execute(mockThing);
     expect(cloudMocks.mockAddDevice).toHaveBeenCalled();
   });
+
+  test('unregisterDevice: should execute interactor correctly when unregister a thing from cloud', async () => {
+    const cloud = new Cloud();
+    const unregisterDevice = new UnregisterDevice(cloud);
+    await unregisterDevice.execute(mockThing);
+    expect(cloudMocks.mockRemoveDevice).toHaveBeenCalled();
+  });
+
+  test('unregisterDevice: should execute interactor without errors when fail to unregister thing from cloud', async () => {
```

Comment:
Suggestion :point_down: 

```suggestion
  test('unregisterDevice: should execute interactor without errors when the unregistration of a thing from the cloud fails', async () => {
```

Reply:
Done!